packages/wonder-blocks-data/src/util/__tests__/memory-cache.test.js

```diff
@@ -306,22 +377,70 @@ describe("MemoryCache", () => {
 
             // Act
             const result = cache.removeAll(fakeHandler);
-            const after = cache.cloneData();
 
             // Assert
             expect(result).toBe(3);
-            expect(after).toStrictEqual({
-                MY_HANDLER: {},
-                OTHER_HANDLER: {
+        });
+    });
+
+    describe("#cloneData", () => {
+        it("should return a copy of the cache data", () => {});
```

Comment:
Are you planning on writing this test in a future PR?

Reply:
Oh, no, it should be written here. Hmm. Good catch, thank you.